pkg/jx/cmd/test_data/step_create_task/command-as-multiline-script/pipeline.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: tekton.dev/v1alpha1
```

Comment:
This file should be removed - I'm guessing it's a lurking artifact from before the `cmd` package got reorganized. =)

Reply:
But it's not getting used in the tests - the one in `pkg/jx/cmd/step/create/test_data` is.